doc/jemalloc.xml.in

```diff
@@ -2623,8 +2629,8 @@ struct extent_hooks_s {
           <literal>r-</literal>
           [<option>--enable-stats</option>]
         </term>
-        <listitem><para>Cumulative number of small allocation requests.
-        </para></listitem>
+        <listitem><para>Cumulative number of allocation requests satisfied by
+        all bin size classes.</para></listitem>
```

Comment:
IMO tcache isn't directly relevant here, and trying to say anything here confuses things.  We could say these are "application-originated" allocation requests, but IMO that's redundant.  Note that I expanded the ```nmalloc``` docs to help differentiate ```nrequests```; if you read the docs for both stats does it still seem confusing?